framework/components/ema-ratings/ema-ratings.twig

```diff
@@ -34,9 +34,10 @@
         {{ _disabled ? 'disabled="disabled"'}}
         {{ i == _default_rating ? 'aria-checked="true" checked="checked"' }}
       />
-      <label class="ema-rating__star" for="ema-rating__{{_name}}--{{i}}">
+      <label class="ema-rating__star" {{ not _disabled ? 'for="ema-rating__' ~ _name ~ '--' ~ i ~ '"': 'disabled="disabled"'}}>
         ★
       </label>
     {% endfor %}
   </div>
+  <span class="ema-rating__feedback ecl-u-ml-s"></span>
```

Comment:
Display of feedback is confusing (pushing the rest of the line). So we will have to alter the display here.
I see several possibilities:
- add some space between rating blocks
- add some space below (between rating and comment field)
- use tooltips instead of plain text

In any case we should prevent content to be pushed each time we hover a star